### PR TITLE
[doc] Update available images

### PR DESCRIPTION
**Pull request by saimidu**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Add PT 1.10.2 to available images, and fix image URIs for TF 2.7 SM DLCs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: available_images.md [hunk 1]
```diff
@@ -87,20 +87,20 @@ You can pin your version by adding the version tag to your URL as follows:
...
-|MXNet 1.9.0   |training	|Yes			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.9.0-cpu-py38-ubuntu20.04-e3		|
-|MXNet 1.9.0  |training	|Yes			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.9.0-gpu-py38-cu112-ubuntu20.04-e3	|
-|MXNet 1.9.0   |inference	|No			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-inference:1.9.0-cpu-py38-ubuntu20.04-e3        |
-|MXNet 1.9.0   |inference	|No			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-inference:1.9.0-gpu-py38-cu112-ubuntu20.04-e3	|
+| Framework        |Job Type	|Horovod Options|CPU/GPU 	|Python Version Options	| Example URL																						                                                                      |
+|------------------|-----------|---------------|-----------|-----------------------|--------------------------------------------------------------------------------------------------------|
+| PyTorch 1.10.2   |training	|No			|CPU 		| 3.8 (py38)			| 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.10.2-cpu-py38-ubuntu20.04-e3		         |
```
Comment (shantanutrip) [inline available_images.md:92]: Thanks for making the change for PT1.10.2 and rectifying the spaces. Shall we add PT1.10.0 to `Prior E3 Framework Container Versions` section beneath?

Comment (shantanutrip) [reply available_images.md:92]: Also, lets close this PR https://github.com/aws/deep-learning-containers/pull/1747/files once we get this merged. Thanks!!

Comment (saimidu) [reply available_images.md:92]: Thank you for opening the PR for PT 1.10.2 before. Didn't get a chance to review it before.

Comment (shantanutrip) [reply available_images.md:92]: Got it, thanks!!

**Review by shantanutrip — APPROVED** [review]

**Pull request merged**